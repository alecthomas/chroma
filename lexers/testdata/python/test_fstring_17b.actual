f"x = {x:+3}"
