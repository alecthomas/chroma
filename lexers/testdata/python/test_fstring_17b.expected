[
  {"type":"LiteralStringAffix","value":"f"},
  {"type":"LiteralStringDouble","value":"\"x = "},
  {"type":"LiteralStringInterpol","value":"{"},
  {"type":"Name","value":"x"},
  {"type":"LiteralStringInterpol","value":":"},
  {"type":"LiteralStringDouble","value":"+3"},
  {"type":"LiteralStringInterpol","value":"}"},
  {"type":"LiteralStringDouble","value":"\""},
  {"type":"Text","value":"\n"}
]
